model/src/main/scala/ch.epfl.scala.index.model/release/LanguageVersion.scala

```diff
@@ -34,8 +34,12 @@ final case class ScalaVersion(version: BinaryVersion) extends LanguageVersion {
 }
 
 final case class DottyVersion(version: BinaryVersion) extends LanguageVersion {
```

Comment:
I think we should rename the class to `Scala3Version`|

Reply:
yes